R/axis.R

```diff
@@ -32,8 +32,8 @@ easy_rotate_x_labels <- function(angle = 90, side = c("left", "middle", "right")
     angle
   } else if (is.character(angle)) {
     switch(angle,
-      startattop = -90,
-      startatbottom = 90
+           startattop = -90,
```

Comment:
Could you please revert this file if there are only unrelated whitespace changes?